UNITY.md

```diff
@@ -0,0 +1,160 @@
+# CAVE Unity Project
+
+## Unity Development
+
+- The origin is the middle of all objects
+- Nested objects will assume the local transform of the parent object
+  - Position of the object itself is still in the parent's space
+- UNITY USES METERS FOR UNITS (CAVE uses ft)
+  - CAVE walls are 8' (96") squares -> 2.4384 meters
+  - 1ft = 0.3048m || 1m = 3.28084ft
+
+### Unity Starter Assets
+
+- Base project uses the "VR Project" template project that Unity provides
+- Delete the Skybox (`Lighting -> Environment -> Skybox Material`)
+  - Set "Environment Lighting" to "Color"
+  - Set "Ambient Color" to white (255, 255, 255)
+- The starter DirectionalLight can be deleted
+- The starter `XRRig` stays as a root object of the project. Keep defaults.
+- The entire project lives inside the `Story` empty game object
+
+### VR Settings
+
+The Unity project settings and XRRig must be adjusted for VR to work:
+
+- The "XR Plugin Management" package should be installed by virtue of the "VR Project" template. **Version 4.2.1**. In the "XR Plug-In Management" section of the Project Settings:
+  - Add the "Oculus Touch Controller Profile" to the `OpenXR -> Interaction Profiles` list
+    - *HTC, Microsoft, Valve, etc support can also be enabled here*
+- The "XR Interaction toolkit" package must be added my name - `com.unity.xr.interaction.toolkit`. **Version 2.0.2**
+  - Download the "Starter Assets" from the samples download. This downloads the needed presets for VR interaction.
+    - I moved the starter assets to the `ExampleAssets/XR InteractionToolkit` folder
+- Add the "XR Controller (Action Based)" script to each controller
+  - "XRI Default Left Controller" preset for the LeftController object
+  - "XRI Default Right Controller" preset for the RightController object
+- Add the "Input Action Manager" script to the XRRig component
+  - Add the "XRI Default Input Actions" preset to the "Action Assets" list
+    - Note that this is what maps the software actions to the different hardware buttons
+
+Check out [this tutorial](https://www.youtube.com/watch?v=5ZBkEYUyBWQ) for more information on VR with Unity and the XR Interaction Toolkit.
+
+**TODO**: How to do ray based interaction? (52)
+
+### MiddleVR Settings
+
+In addition to the Unity XRRig, a root MiddleVR package needs to be added in order for the project to work with MiddleVR and the CAVE.
+
+- More info to come.
+
+## PlacementRoot
+
+Each `<Placement>` in `PlacementRoot` is a Quad. PlacementRoot is the same for all projects:
+
+```xml
+<PlacementRoot>
```

Comment:
Is this in a file somewhere in a repo? If so, would be good to just add a file path to it in the readme rather than copying and pasting it all here.